pkgs/servers/radicale/default.nix

```diff
@@ -22,6 +22,7 @@ python3.pkgs.buildPythonApplication rec {
     vobject
     python-dateutil
     passlib
+    setuptools
```

Comment:
this should be in nativeBuildInputs (setuptools is only needed during initial build, think similar to cmake)

Reply:
Yeah usually that's the case, but here this is apparently needed at runtime, putting it in `nativeBuildInputs` doesn't fix the issue.

Reply:
if it's a python2 namespace package, then it will also need it at runtime. Interesting that it wants to grab an entry point.